.goreleaser.yml

```diff
@@ -0,0 +1,31 @@
+env:
+  - GO111MODULE=on
+before:
+  hooks:
+    - go mod tidy
```

Comment:
are `tidy` and `verify` commands really necessary?
if they are expected to change the repo contents, would it be better if our pipeline had a check in a separate stage - to check that the repo files do not change after tidy and verify?

Reply:
they're not expected to change the repository's content, it's just a check to make sure everything is fine.

Reply:
`go mod tidy` exits with 0 status code even if it updates the go.mod. so it will silently fix the files.